### ISSUE TITLE
Dataset Import UI elements

### ISSUE THREAD

**Issue by jpggvilaca** [open-edge-platform/training_extensions#5395]
Similar experience from Geti Classic
 
* Button to IMPORT dataset
* Dialog to prepare dataset (map labels, etc)
* Visualisation of the importing job
* API integration
* Tests

**Comment by leoll2** [comment]
W.r.t. Geti Classic, there are a few differences in the user flow when importing a dataset:
- In the prepare phase, the user can filter which labels and subsets _of the dataset_ to import.
  - Example 1: import dataset items assigned to the training set, but not the validation/testing ones.
  - Example 2: import only annotations of 'cat' and 'dog', but not 'elephant'.
-  When importing to an existing project, the user can map labels and subsets of the dataset to the project's one
  - Example 3: import the items with subset 'testing' (in the dataset) into the validation set of the project
  - Example 4: map labels 'cat' and 'dog' to 'pet', and 'lion' to 'wild animal'.
- The user can choose to include/exclude unannotated items present in the dataset
- Warning are removed
